toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -23,39 +28,97 @@ get_lkg() {
 }
 
 check_for_modified_manifests() {
+    auto=$1
+    force=$2
```

Comment:
Consistency nit-pick:
```suggestion
    local auto=$1
    local force=$2
```

Or remove `local` from the other function.